packages/extract-json/README.md

```diff
@@ -17,42 +17,39 @@ yarn add @iota/extract-json
 
 ## API Reference
 
-    <a name="module_extract-json..extractJson"></a>
+    <a name="module_extract-json.extractJson"></a>
```

Comment:
```suggestion
<a name="module_extract-json.extractJson"></a>
```

Removing these spaces should fix it for the time being, and then we can open an issue at `jsdoc-to-markdown`. We shouldn't block the entire PR over a small issue imo

Reply:
Created an [issue](https://github.com/jsdoc2md/jsdoc-to-markdown/issues/217). I would prefer to not make this manual change right now, because any automated changes to docs will revert this change.